src/resources/config.json

```diff
@@ -18,6 +18,6 @@
     "defaultAgg": "mean",
     "allowDownloads": true,
     "polygonFillOpacity": 0.2,
-    "mapProjection": "EPSG:4326"
+    "mapProjection": "EPSG:4326",
```

Comment:
```suggestion
    "mapProjection": "EPSG:4326"
```

The comma needs to be removed, otherwise it leads to an error: 

![image](https://user-images.githubusercontent.com/23457217/168539550-0c8c9ffa-7fc7-4c81-8ccf-0af4eb15faa1.png)

Reply:
Oops, this change has nothing to with the PR! Will fix.